corgi/web/templates/component_manifest.json

```
@@ -79,4 +86,8 @@
       "relatedSpdxElement": "SPDXRef-{{obj.uuid}}",
       "relationshipType": "GENERATES",
       "spdxElementId": "SPDXRef-{{node_id}}"
-    },{% endfor %}{% endif %}{% endblock relationships %}
\ No newline at end of file
+    },{% endfor %}{% endif %}    {
```

Please take care not to add or remove whitespace anywhere in the template - those changes will directly end up in the final manifest. You may want to generate a few stream and component manifests using the new code here, then compare them with manifests we've already generated in stage, and confirm there are no whitespace changes.
```suggestion
    },{% endfor %}{% endif %}
    {
```

---

updated.